src/lib/storage/index/adaptive_radix_tree/adaptive_radix_tree_index.cpp

```diff
@@ -17,6 +17,12 @@
 
 namespace opossum {
 
+float AdaptiveRadixTreeIndex::predict_memory_consumption(ChunkOffset row_count, ChunkOffset value_count,
+                                                         uint32_t value_bytes) {
+  // ToDo(anyone): If you use this index in combination with the Tuning subsystem, you need to properly implement this.
+  return std::numeric_limits<float>::quiet_NaN();
```

Comment:
I would be fine with making this fail. Whoever wants to use the Tuner with the ART should implement this.

Reply:
done, it fails now